tpu/templates/tools/parser_add.py

```diff
@@ -0,0 +1,67 @@
+import glob
```

Comment:
Add information about what this file does at the top.

Reply:
glob is one of python's built in modules.

RedBaron is more problematic but to this point I haven't found a good substitute.